### PR TITLE
Add pcre and pcre-devel as unwanted

### PR DESCRIPTION
**Pull request by ljavorsk**
Pcre project has been deprecated and unsupported by upstream since Fedora 38 Fedora Change: https://fedoraproject.org/wiki/Changes/PcreDeprecation

### CODE REVIEW

File: configs/sst_cs_apps-apps-unsorted.yaml [hunk 1]
```diff
@@ -10,8 +10,6 @@ data:
   - time
   - pcre2
   - pcre2-devel
-  - pcre
```
Comment (jwboyer) [inline configs/sst_cs_apps-apps-unsorted.yaml:13]: This is included in CentOS Stream 9 already.  You can't drop it from this file with the `c9s` label included.  You likely want to create a new eln specific workload and unwanted file instead.

**Comment by ljavorsk** [comment]
Did I fix it correctly @jwboyer ?

**Comment by yselkowitz** [comment]
No, because it is still listed as required in `sst_cs_apps-apps-unsorted.yaml`.  In addition to the eln unwanted file, the c9s and eln configs for apps-unsorted need to be split into separate files, with the pcre/pcre-devel packages removed from the eln config.

**Comment by ljavorsk** [comment]
I've split the files, can you check them now, please?

Sorry for the confusion, it's my first time with content resolver

**Pull request merged**